packages/http/src/mocker/HttpMocker.ts

```diff
@@ -72,6 +72,7 @@ export class HttpMocker implements IMocker<IHttpOperation, IHttpRequest, IHttpCo
         'Content-type': negotiationResult.mediaType,
       },
       body: mockedBody,
+      responseType: '', // todo(Vincenzo): to be implemented
```

Comment:
I have no idea what to put here in a mocked response to be honest.
The spec gives some value that I will probably never going to use; it does not even support XML. 

@philsturgeon